### DISCUSSION TITLE
Adding Import Functionality to Editor

### DISCUSSION THREAD

**Discussion by GaikwadShubham** [GrapesJS/grapesjs#3854] [Q&A]
Hi @artf , I am looking into adding the import code functionality into my code. 
![image](https://user-images.githubusercontent.com/78800459/136320743-ef229b8c-073f-4737-9048-5c154014001c.png)

I search a lot for import plugin (as like [export plugin](https://github.com/artf/grapesjs-plugin-export)) but I didn't found it. If any of such plugin is created can you please help me with it.
If not, can you redirect me to the code which I need to include in my code base to use this functionality. 

Thanks for creating this awesome framework !!

**Comment by artf** [comment]
You can copy the implementation made for the demo from here https://github.com/artf/grapesjs-preset-webpage/blob/master/src/commands/openImport.js
